chainstate/src/detail/tx_verification_strategy/default_strategy.rs

```diff
@@ -81,9 +41,8 @@ impl Default for DefaultTransactionVerificationStrategy {
 }
 
 impl TransactionVerificationStrategy for DefaultTransactionVerificationStrategy {
-    fn connect_block<'a, H, S, M>(
+    fn connect_block<'a, H, S>(
         &self,
-        tx_verifier_maker: M,
```

Comment:
Why did you remove this?

Reply:
To implement disposable strategy I need to create different verifiers that could be modified as well as not. The function limits this to read only `S`.

Reply:
No, I'm wrong. It was the case but it was fixed by changes in chainstate/tx-verifier/src/transaction_verifier/hierarchy.rs lline 108.
Will revert.